script/developer/14.0/update/DeveloperUpdateCheck.php

```diff
@@ -0,0 +1,105 @@
+<?php
+  declare(strict_types=1);
+
+  namespace Keyman\Site\com\keyman\api;
+
+  require_once __DIR__ . '/../../../../tools/autoload.php';
+
+  use Keyman\Site\com\keyman\api\DownloadsApi;
+  use Keyman\Site\Common\KeymanHosts;
+
+  class DeveloperUpdateCheck {
+    const SETUP_REGEX = '/^keymandeveloper-.+\.exe/';
+
+    private $isManual;
+    private $currentTime; // used mainly for unit testing
+
+    public function execute($mssql, $tier, $appVersion, $isManual, $currentTime = null) {
+
+      $this->mssql = $mssql;
+      $this->currentTime = $currentTime;
+
+      $isUpdate = empty($isUpdate) ? 0 : 1;
+      $this->isManual = !empty($isManual);
+
+      $developer_update = [];
+      $developer_update['developer'] = $this->BuildKeymanDesktopVersionResponse($tier, $appVersion, self::SETUP_REGEX);
+
+      return json_encode($developer_update, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);
+    }
+
+    private function BuildKeymanDesktopVersionResponse($tier, $InstalledVersion, $regex) {
```

Comment:
Should this be `BuildKeymanDeveloperVersionResponse`? It appears to be checking the Developer version.